reference-tests/src/test/java/net/consensys/linea/BlockchainReferenceTestTools.java

```diff
@@ -105,6 +119,16 @@ public class BlockchainReferenceTestTools {
     // Absurd amount of gas, doesn't run in parallel.
```

Comment:
I imagine that the CALL50000 could prove problematic with the MemoryRange implementation we currently have @letypequividelespoubelles ^^ If we have to keep 50k snapshots of memory ^^ I mean, it's not an issue. But I wonder whether the tests that @amkCha was able to run again on that 64GB machine would crash with our current tracer.